public/index.html

```diff
@@ -31,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Gurkult</title>
+    <title>React App</title>
```

Comment:
It appears intentional changes to the meta were reverted here. We need this to remain accurate so this will embed properly on discord.